test/chpldoc/globals/arrayTypeAlias.doc.good

```diff
@@ -17,7 +17,7 @@ or
 
    import arrayTypeAlias.doc;
 
-.. type:: type t = [1..10] int;
+.. type:: type t = [1..10] int
```

Comment:
In running tests, it looks like a similar change is needed for chpldoc/globals/arrayLiteral.doc.chpl

Reply:
I'll change this and send it rn.